packages/cypress-mock-openapi/cypress.env.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What's the reason for the env being both here and in cypress.json?

Reply:
🤔 I really don't know ! I just copied the files. Did Cypress copy data? I will have to check.

Reply:
Don't know what happened. I removed the lines, ran Cypress again, and that stayed that way 🤷🏻‍♂️

Reply:
I guess it's Cypress behaviour so we can leave it in 🤷‍♀️

Reply:
No I mean I don't know why these lines were copied in the first place, as apparently Cypress does nothing. I updated the PR so that these lines are only in `cypress.env.json`, not in `cypress.json`, and all is working well.